packages/user/src/supertokens/utils/isRoleExists.ts

```diff
@@ -1,5 +1,8 @@
 import UserRoles from "supertokens-node/recipe/userroles";
 
+/**
+ * @deprecated Use areAllRolesExist instead.
+ */
 const isRoleExists = async (role: string): Promise<boolean> => {
```

Comment:
I think we do not have to deprecate it if anyone want they can check it for a single role..